src/unitxt/catalog/tasks/llm_as_judge/model_response_assessment.json

```diff
@@ -0,0 +1,13 @@
+{
+    "type": "form_task",
+    "inputs": [
+        "question",
+        "prediction"
+    ],
+    "outputs": [
+        "output"
```

Comment:
rating_label

Reply:
prediction_type: "Tuple[float,str]"

Reply:
float is between 0 and 1

Reply:
@eladven - I thought about it more.  Adding the explanation would complicate this task API.  I suggest the prediction_type will be just float.  It will require dedicated metric pipeline to use standard metrics like spearman.   If we want, in the future we can add model_ressponse_assessment_with_explanation task.